### DISCUSSION TITLE
Disable/override localizations?

### DISCUSSION THREAD

**Discussion by elenakrittik** [microsoft/pyright#6916] [Q&A] [answered]
I recently updated pyright from an ancient version to 1.344 and now i get error messages in my native language instead of english. My system (windows 10) is set up to use english locale for text and my native one for date/time. I don't know how pyright detects the locale to use and not sure whether this is worth opening a bug ticket, but is there at least a cli flag or something that i can use to force english locale?

**Comment by erictraut** [comment, marked as answer]
Thanks for the question. I realized that we hadn't documented a way to override the default locale. I just added [new documentation](https://microsoft.github.io/pyright/#/configuration?id=locale-configuration) that explains how to do this.
